modules/claims_api/app/sidekiq/claims_api/reporting_base.rb

```diff
@@ -51,21 +51,6 @@ def claims_totals
       end
     end
 
-    def monthly_claims_totals
-      @claims_consumers.map do |cid|
-        counts = ClaimsApi::AutoEstablishedClaim.where(created_at: @from..@to, cid:).group(:status).count
-        totals = counts.sum { |_k, v| v }.to_f
-
-        if totals.positive?
-          consumer_name = ClaimsApi::CidMapper.new(cid:).name
-          {
-            consumer_name => counts.merge(totals:)
-                                   .deep_symbolize_keys
-          }
-        end
-      end
-    end
-
```

Comment:
Moving this method to the `ReportMonthlySubmissions` class since we don't use it anywhere else.

Also removing the N+1 query.